### PR TITLE
bash: read $PROMPT_COMMAND without interpreting escapes (raw mode)

### LINKED ISSUE #41: Issue with `pazi init bash` output
```
-bash: PROMPT_COMMAND: line 1: unexpected EOF while looking for matching `"'
-bash: PROMPT_COMMAND: line 2: syntax error: unexpected end of file
```

Looks like my PROMPT_COMMAND has something in it already from gnome-terminal and evaling pazi's init code is breaking it somewhat.

Without `$(pazi init bash)` in my .bashrc:

```
[ilianaw@2b ~]$ echo $PROMPT_COMMAND
printf "\033k%s@%s:%s\033\\" "${USER}" "${HOSTNAME%%.*}" "${PWD/#$HOME/\~}"
```

With:

```
[ilianaw@2b ~]$ echo $PROMPT_COMMAND                 
printf "033k%s@%s:%s033\" "${USER}" "${HOSTNAME%%.*}" "${PWD/#$HOME/~}"; __pazi_add_dir;                  
```

### PR DESCRIPTION
**Pull request by LinuxMercedes**
Fixes #41.

### CODE REVIEW

**Comment by euank** [comment]
Wanna add an integ test specifically for this?

File: src/shells/bash.rs [hunk 1]
```diff
@@ -19,7 +19,7 @@ __pazi_add_dir() {
 if [[ -z "${PROMPT_COMMAND}" ]]; then
     PROMPT_COMMAND="__pazi_add_dir;"
 else
-    PROMPT_COMMAND="$(read newVal <<<"$PROMPT_COMMAND"; echo "${newVal%;}; __pazi_add_dir;")"
+    PROMPT_COMMAND="$(read -r newVal <<<"$PROMPT_COMMAND"; echo "${newVal%;}; __pazi_add_dir;")"
```
Comment (euank) [inline src/shells/bash.rs]: I think I'd rather switch to:

```
PROMPT_COMMAND="__pazi_add_dir;\$PROMPT_COMMAND"
```

And cite https://github.com/clvv/fasd/blob/90b531a5daaa545c74c7d98974b54cbdb92659fc/fasd#L132-L136 as inspiration.

The stack overflow comment claiming this was safe is clearly wrong, and I'd trust fasd's method to have been more battle-tested by now.

**Comment by ilianaw** [comment]
bfa1ce0a7a7b54f5ea33874c0866ce421913edb5 fixes me but I agree with @euank's comment.

File: src/shells/bash.rs [hunk 2]
```diff
@@ -7,6 +7,7 @@ impl Shell for Bash {
         // ty to mklement0 for this suggested append method:
         // https://stackoverflow.com/questions/3276247/is-there-a-hook-in-bash-to-find-out-when-the-cwd-changes#comment35222599_3276280
```
Comment (euank) [inline src/shells/bash.rs]: nix this comment as well since it's no longer relevant

**Pull request merged**